app/src/store/__tests__/token.test.ts

```diff
@@ -87,12 +88,35 @@ describe('token store', () => {
     });
     fetchMock.mockResolvedValueOnce({
       ok: false,
+      text: jest.fn().mockRejectedValueOnce({}),
     });
     await store.dispatch(user.login());
     await store.dispatch(token.getAccessToken());
     expect(store.getState().voice.accessToken).toEqual({
       reason: 'TOKEN_RESPONSE_NOT_OK',
       status: 'rejected',
+      error: miniSerializeError(null),
+    });
+  });
+
+  it('rejects if "TOKEN_RESPONSE_NOT_OK" and status is fulfilled', async () => {
+    jest.spyOn(auth0, 'authorize').mockResolvedValueOnce({
+      accessToken: 'test token',
+      idToken: 'test id token',
+    });
+    const errorMessage = 'foobar';
```

Comment:
Let's use an actual error object. We can pass `"foobar"` as the message for the error but it's closer to the real thing if it's in an error object.